### ISSUE TITLE
More sophisticated support for massviews 

### ISSUE THREAD

**Issue by Ladsgroup** [MusikAnimal/pageviews#111]
Hey,
It would be great to support such cases:

1. Results from quarry
2. Toggle main page of talk pages (to support English Wikipedia system of Wikiprojects. For example. If I want to check [High-importance medicine articles](https://en.wikipedia.org/wiki/Category:High-importance_medicine_articles). It gives me talk pages.
3. Support template transclusions. For example, If I want to see view of pages that used Template:Infobox scientist.

**Comment by MusikAnimal** [comment]
Awesome recommendations! Especially #3, great idea :)

**Comment by MusikAnimal** [comment]
Going to list this out and develop/deploy one at a time (checked means deployed):

- [ ] Results from quarry
- [ ] Toggle for main page of talk pages
- [ ] Template transclusions
- [ ] Text field to manually enter pages

**Comment by MusikAnimal** [comment]
@Ladsgroup Could you give me a specific use-case or example of Quarry results? Should we just assume the first column of the result set are the page names?

**Comment by Ladsgroup** [comment]
I think we should note in documentations that the first column should be page_title but otherwise I think that's okay. I can't think of any use cases but it's very likely to generate list of pages for a given purpose via quarry :) thanks for your work on this

**Comment by MusikAnimal** [comment]
@Ladsgroup Deployed :) From my comment above I was going to also add a text field to manually enter pages, but this is a lot of work for little benefit. If users want to create a list they should just use PagePile so the list can be retained.

Anyway I've fulfilled your request, I believe. Note Quarry results are also capped at 500 pages (for the time being) in the interest of performance. Also using Quarry doesn't work in Safari right now, I will create a new issue for that.

Let me know if there's anything else we can do to improve the tool! Closing.
